packages/customize-widgets/src/components/inserter/index.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * WordPress dependencies
+ */
+import { useRef, createPortal } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { __experimentalLibrary as Library } from '@wordpress/block-editor';
+import { Button } from '@wordpress/components';
+import { useInstanceId } from '@wordpress/compose';
+
+/**
+ * External dependencies
+ */
+import { Dialog, DialogBackdrop } from 'reakit/Dialog';
+
+function useTargetDOM() {
+	const target = useRef();
+	if ( ! target.current ) {
+		target.current = document.getElementById( 'widgets-left' );
+	}
+	return target.current;
+}
+
+function Inserter( { ...props } ) {
+	const target = useTargetDOM();
+	const inserterTitleId = useInstanceId(
+		useRef(),
+		'customize-widget-layout__inserter-panel-title'
+	);
+
+	return createPortal(
+		<DialogBackdrop
+			className="customize-widgets-layout__inserter-panel-backdrop"
+			{ ...props }
+		>
+			<Dialog
+				{ ...props }
+				// Unset the "dialog" role to make it possible to be close the quick inserter on focus.
```

Comment:
Is this a leftover from a copy/paste? As far as I can tell, this code doesn't have anything to do with the Quick Inserter.

Reply:
It's actually intended here. When clicking "browse all" in the quick inserter, the inserter opened and being focused. However, the quick inserter won't be closed if the `activeElement` is inside an element with `role="dialog"`. Perhaps I should make the comment clearer (also spotted a type 😅 ).

Furthermore, I think this is more like a "popover" rather than a "dialog", hence I think it makes sense to not use the dialog role? Would appreciate some reviews from the a11y experts.

Reply:
Ahh gotcha. Yes please update the comment ❤️